src/openrct2/ride/TrackDesign.cpp

```diff
@@ -2148,7 +2148,7 @@ static void TrackDesignPreviewClearMap()
 {
     auto numTiles = MAXIMUM_MAP_SIZE_TECHNICAL * MAXIMUM_MAP_SIZE_TECHNICAL;
 
-    gMapSize = 256;
+    gMapSize = { 256, 256 };
```

Comment:
This _used_ to be the maximum map size. Since it’s ludicrously big for a track design preview already, I propose creating a constant for it (e.g. TRACK_DESIGN_PREVIEW_MAP_SIZE).